tests/transforms/test_experimental/test_transform_dispatcher.py

```diff
@@ -486,7 +486,10 @@ def comb_postproc(results: TensorLike, fn1: Callable, fn2: Callable):
 
         # Create a simple device and tape
         tmp_dev = qml.device("default.qubit", wires=3)
-        H = qml.PauliY(2) @ qml.PauliZ(1) + 0.5 * qml.PauliZ(2) + qml.PauliZ(1)
+
+        H = qml.Hamiltonian(
```

Comment:
Why are we making these changes from using `@` and `+` to explicitly calling `qml.Hamiltonian`?